core/upgrade/upgrades.php

```diff
@@ -537,12 +538,13 @@ public static function _v_2_6_0_fa4_migration_flag() {
 	 *
 	 * @return mixed
 	 */
-	public static function _migrate_icon_fa4_value( $element, $args ) {
+	public static function _migrate_fa_icon_values( $element, &$args ) {
```

Comment:
Changing public function, will it be a problem?
What if someone using it?

Reply:
No-one should ever rely on migration functions inside the Upgrades class.
They are intended for internal use only.